### PR TITLE
[Hub Generated] Review request for Microsoft.SecurityInsights to add version preview/2019-01-01-preview

### PR DESCRIPTION
**Pull request by hassanchawiche**
This is a PR generated at [OpenAPI Hub](https://aka.ms/openapihub). You can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/hassanchawiche/azure-rest-api-specs/dev-securityinsights-Microsoft.SecurityInsights-2019-01-01-preview...master/).

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “**WaitForARMFeedback**” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json [hunk 1]
```diff
@@ -3423,51 +3423,6 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/providers/{operationalInsightsResourceProvider}/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/watchlists": {
```
Comment (chiragg4u) [inline specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json:3426]: >watchlists [](start = 149, length = 10)

Removing and API from existing version is a breaking change.

**Comment by chiragg4u** [comment]
As identified in the automated checks there are breaking changes, Please follow Azure Breaking changes approval request form as defined @ [Breaking Change Process](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37673/Retirements-and-Breaking-Changes)

**Review by JeffreyRichter — APPROVED** [review]

**Review by majastrz — APPROVED** [review]
Breaking changes board approved the breaking change and the API looks fine as well from ARM perspective. Signed off from ARM side.

That said, you need to fix your linter failures.

**Comment by njuCZ** [comment]
@hassanchawiche Have you gone thought the [Breaking Change Process](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37673/Retirements-and-Breaking-Changes) ?
since this is a preview version for more than 1 year, is there a deadline for it to become stable?

**Comment by hassanchawiche** [comment]
> @hassanchawiche Have you gone thought the [Breaking Change Process](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37673/Retirements-and-Breaking-Changes) ?
> since this is a preview version for more than 1 year, is there a deadline for it to become stable?

Yes, they approved my changes by Jeffrey Richter. This is the approved work item : https://msazure.visualstudio.com/One/_workitems/edit/8534867 

As for the preview version, we do have one stable (2020-01-01). Watchlist is a new feature, and it's currently in preview.

**Comment by njuCZ** [comment]
@hassanchawiche since this PR also remove and rename some fields, this also belongs to breaking change, we still needs the approval.

**Comment by njuCZ** [comment]
since this PR is already approved by JeffreyRichter, I will merge it

**Review by njuCZ — APPROVED** [review]

**Pull request merged**